### PR TITLE
update exocom

### PR DESCRIPTION
**Pull request by charlierudolph**
released some new versions of the templates and now remove the need for the EXOCOM_PORT env variable

### CODE REVIEW

File: features/tutorial.feature [hunk 1]
```diff
@@ -182,16 +182,20 @@ Feature: Following the tutorial
...
 
         index(req, res) {
-          this.send('todo.list', {}, (todos) => {
-            res.render('index', {todos})
+          this.send('todo.list', {}, (messageName, payload) => {
+            if (messageName === 'todo.listing') {
+              res.render('index', {todos: payload})
+            } else {
+              res.sendStatus(500)
+            }
```
Comment (hugoesthere) [inline features/tutorial.feature:190]: Were these changes meant to be included as well?

**Review by hugoesthere — APPROVED** [review]
Looks good, `exocom.tf` will need to be updated as well. This line can be removed: https://github.com/Originate/exosphere/blob/master/src/terraform/templates/exocom.tf#L30 since the default value when one isn't provided is `80`

File: features/tutorial.feature (hunk 1 above)
Comment (charlierudolph) [reply features/tutorial.feature:190]: Yeah. They were a result of a change in exorelay

**Comment by charlierudolph** [comment]
@hugobho want to take one last look?

File: src/terraform/modules/aws/custom/exocom/exocom-service/exocom-service.tf [hunk 2]
```diff
@@ -2,7 +2,6 @@ module "task_definition" {
   source = "./exocom-task-definition"
 
   command               = "${var.command}"
-  container_port        = "${var.container_port}"
```
Comment (hugoesthere) [inline src/terraform/modules/aws/custom/exocom/exocom-service/exocom-service.tf]: Let's keep this variable in, that way we are in line with behavior on Exocom: container ports come online at port 80 by default but can be overridden if one is passed down. Removing this line will prevent us from passing one down at all. Just removing `container_port = "3100"` in `exocom.tf` and in the corresponding test is sufficient.

**Review by hugoesthere — APPROVED** [review]
Good to go after that comment is addressed.

**Pull request merged**